x-pack/docs/en/security/fips-140-compliance.asciidoc

```diff
@@ -27,8 +27,8 @@ For {es}, adherence to FIPS 140-2 is ensured by
 [discrete]
 === Upgrade considerations
 
-[IMPORTANT] 
```

Comment:
Intellij keeps auto-formatting this. I'm assuming it's fine to drop them trailing spaces